lib/screens/explore_screens/vote_screens/vote/vote_screen.dart

```diff
@@ -23,12 +23,14 @@ class VoteScreen extends StatelessWidget {
         child: Scaffold(
           appBar: AppBar(
             actions: [
-              IconButton(
-                onPressed: () {
-                  NavigationService.of(context).navigateTo(Routes.delegate);
-                },
-                icon: SvgPicture.asset('assets/images/explore/delegate.svg'),
-              ),
+              if (VoteBloc().state.shouldShowDelegateIcon)
```

Comment:
is this how we access blocs normally? VoteBloc().

Reply:
Nope, he needs wrap IconButton with `BlocBuilder<VoteBloc, VoteState>` and the in the return a ternary
`state.shouldShowDelegateIcon? IconButton : SizedBox.shrink();`

Reply:
fix